docs/docsite/rst/vmware/scenario_remove_vm.rst

```diff
@@ -0,0 +1,127 @@
+.. _vmware_guest_remove_virtual_machine:
+
+*****************************************
+Remove an existing VMware virtual machine
+*****************************************
+
+.. contents:: Topics
+
+Introduction
+============
+
+This guide will show you how to utilize Ansible to remove an existing VMware virtual machine.
+
+Scenario Requirements
+=====================
+
+* Software
+
+    * Ansible 2.5 or later must be installed.
+
+    * The Python module ``Pyvmomi`` must be installed on the Ansible (or Target host if not executing against localhost).
+
+    * Installing the latest ``Pyvmomi`` via pip is recommended [as the OS packages are usually out of date and incompatible].
+
+* Hardware
+
+    * At least one standalone ESXi server or
+
+    * vCenter Server with at least one ESXi server
+
+* Access / Credentials
+
+    * Ansible (or the target server) must have network access to the either vCenter server or the ESXi server
+
+    * Username and Password for vCenter or ESXi server
+
+Assumptions
+===========
+
+- All variable names and VMware object names are case sensitive.
+- You need to use Python 2.7.9 version in order to use ``validate_certs`` option, as this version is capable of changing the SSL verification behaviours.
+
+Caveats
+=======
+
+- Hosts in the ESXi cluster must have access to the datastore that the template resides on.
+- ``vmware_guest`` module tries to mimick VMware Web UI and workflow, so the virtual machine must be in powered off state in order to remove it from the VMware inventory.
+
```

Comment:
Could the three sections of "things you need to have or to know" - Caveats, Assumptions, and Scenario Requirements - be merged into one section called Prerequisites?

Reply:
Reading this again, I can see a good argument for separating the Requirements from the other two sections. I think line 46, about the server having access to templates, belongs under Access/Credentials in the Requirements section. 

Between the Caveats and the Assumptions, there's not as clear of a line. For example, the fact that variable names are case sensitive seems like a caveat rather than an assumption - at least, I think we're saying, "Look out, variables are case sensitive, if you forget that it could trip you up," not "We're assuming case-sensitive variable values in our playbook, but that's not necessarily the case in your environment."

How would you feel about moving line 46 up, then merging the remaining three items under Assumptions and Caveats into a single section? I'd probably keep the heading Caveats for it.